### PR TITLE
Typo fixes

### PR DESCRIPTION
**Pull request by jhadvig**
@kargakis PTAL

### CODE REVIEW

**Comment by kargakis** [comment]
both forms are correct I think

File: pkg/build/api/v1/types.go [hunk 1]
```diff
@@ -55,7 +55,7 @@ type BuildStatus struct {
 	Phase BuildPhase `json:"phase" description:"observed point in the build lifecycle"`
 
 	// Cancelled describes if a cancelling event was triggered for the build.
-	Cancelled bool `json:"cancelled,omitempty" description:"describes if a canceling event was triggered for the build"`
+	Cancelled bool `json:"cancelled,omitempty" description:"describes if a cancelling event was triggered for the build"`
```
Comment (kargakis) [inline pkg/build/api/v1/types.go]: You most probably need to update swagger for this change

./hack/update-generated-swagger-spec.sh

**Comment by jhadvig** [comment]
more or less, but we use the double L in our code + in the printed messages

**Comment by jhadvig** [comment]
@kargakis swagger updated. Thx for noticing :)

**Comment by kargakis** [comment]
> more or less, but we use the double L in our code + in the printed messages

ok, if it's decided between a {special,shady} committee behind closed doors then it looks good to me:)

cc: @fabianofranz @jwforres

File: api/swagger-spec/oapi-v1.json [hunk 2]
```diff
@@ -15305,7 +15305,7 @@
      },
      "cancelled": {
       "type": "boolean",
-      "description": "describes if a canceling event was triggered for the build"
+      "description": "describes if a cancelling event was triggered for the build"
```
Comment (jwforres) [inline api/swagger-spec/oapi-v1.json]: shouldn't this be "describes if a cancel event was triggered for the build"

File: pkg/build/api/v1/types.go (hunk 1 above)
Comment (jwforres) [inline pkg/build/api/v1/types.go]: same thing... should be "cancel" not "cancelling"

**Comment by jwforres** [comment]
and :+1:  for consistency for single vs double L.... even though double L looks very wrong to me, there is no way we are getting the api enumeration value changed at this point, and the single L is only accepted in american english anyway - http://grammarist.com/spelling/cancel/

**Comment by jhadvig** [comment]
@jwforres to be honest I was surprised myself that the double L version was the "more" correct one.  Also I've changed the `description` as suggested. Thanks

**Comment by jhadvig** [comment]
@jwforres is there anything else tbd ?

**Comment by jwforres** [comment]
[merge]

**Comment by jhadvig** [comment]
@jwforres please re-merge. Failed due to flaky tests.

**Comment by jwforres** [comment]
opened flake issue https://github.com/openshift/origin/issues/6149

[merge]

**Comment by jwforres** [comment]
[merge]

failure added to https://github.com/openshift/origin/issues/6121

**Pull request merged**